packages/app/src/app/pages/Dashboard/Components/Folder/FolderCard.tsx

```diff
@@ -57,32 +57,38 @@ export const FolderCard: React.FC<FolderItemComponentProps> = ({
     })}
   >
     <Stack
-      as="div"
-      justify="center"
-      align="center"
-      ref={thumbnailRef}
-      css={css({
-        height: 160,
-        borderStyle: 'solid',
-        borderWidth: 0,
-        borderBottomWidth: 1,
-        borderColor: 'grays.500',
-        backgroundColor: 'grays.600',
-      })}
+      justify="right"
+      paddingTop={5}
+      paddingX={5}
+      css={{ position: 'absolute', width: '100%' }}
     >
-      <svg width={56} height={49} fill="none" viewBox="0 0 56 49">
-        <path
-          fill="#6CC7F6"
-          d="M20.721 0H1.591A1.59 1.59 0 000 1.59v45.82C0 48.287.712 49 1.59 49h52.82A1.59 1.59 0 0056 47.41V7.607a1.59 1.59 0 00-1.59-1.59H28L21.788.41A1.59 1.59 0 0020.72 0z"
+      {!isNewFolder ? (
+        <IconButton
+          name="more"
+          size={9}
+          title="Folder actions"
+          onClick={onContextMenu}
         />
-      </svg>
+      ) : null}
     </Stack>
+
     <Stack
-      justify="space-between"
+      justify="center"
       align="center"
-      marginLeft={4}
-      css={{ minHeight: 26 }}
+      direction="vertical"
+      gap={1}
+      css={{ flexGrow: 1, textAlign: 'center' }}
     >
+      <Stack paddingBottom={4}>
+        <svg width={28} height={25} fill="none" viewBox="0 0 28 25">
```

Comment:
We could add an `aria-hidden="true"` here since this icon is not meaningful to the card context.